### ISSUE TITLE
Explore: Explore button on datasource list

### ISSUE THREAD

**Issue by davkal** [grafana/grafana#19348]
It should be straight-forward to go from a datasource page to Explore, with the datasource preselected.

**Comment by ivanahuckova** [comment]
I am moving this to backlog features, as no-one is currently working on this.

**Comment by davkal** [comment]
@jessover9000 could we have a mockup for this?

**Comment by jessover9000** [comment]
@davkal for the data source list, the new card design is here: https://www.figma.com/file/ITCsaXtsANM17oRtU6GQS9/Component-Playground?node-id=712%3A0
![405 data source list](https://user-images.githubusercontent.com/53754871/98820414-6d8d3b80-242e-11eb-908a-0d567569e02c.png)
![405b data source list narrow](https://user-images.githubusercontent.com/53754871/98820418-70882c00-242e-11eb-8b65-59b770d174e9.png)

That is what you're looking for, right? @Clarity-89 has already implemented a basic card component (https://github.com/grafana/grafana/pull/28216), maybe you can build on top of that for the data source list card design :)

**Comment by Clarity-89** [comment]
@davkal the api for the Card component isn't stable yet and we're working on refactoring it: https://github.com/grafana/grafana/issues/28591.

**Comment by gelicia** [comment]
Waiting on UX to think through solutions on card lists like what we have here. It might be worth thinking through this as a wider design system issue.

**Comment by gelicia** [comment]
We talked about this in a UX design session meeting. Long story short, this section will be revisited with a redesign of a related feature but that may take a while to be complete. In the meantime, we are going to only show the explore button and design it to be similar to a folder management view that is comparable in functionality. It won't be a perfect solution but it will give people this feature until more dedicated resources can revisit the actions available from the data source cards.

**Comment by gelicia** [comment]
Brought this up with the team working on the redesign that involves datasources. The first iteration of the redesign will not bring in the explore button to the datasource list - that iteration is focused more on combining datasources and cloud integrations. The next iteration after that will be about improving the usability of connections, including adding this button so users can jump into their datasource easier.

**Comment by teddyba** [comment]
Hello! Adding an update to this ticket: 

We tested this concept in [Connections research](https://docs.google.com/presentation/d/1QLIOC40OfZxtgOEiiQePy67d91XBdjUG_OmG-yOq2mo/edit#slide=id.g1401d0c8e76_0_81) (check out slide 40) and it tested extremely well. The majority of the users we talked to wanted to further test the data source connection immediately after saving and testing it from the data source configuration page. Users indicated that they would further test the connection by either writing a query in Explore OR creating a dashboard panel. Users who were new to Grafana and/or new to observability or viz/monitoring tools were more likely to go create a dashboard panel than go to Explore. 

The Connections team is prioritizing the effort to include shortcuts to either go to Explore or create a new dashboard panel directly from the data source card. Rough timeline will be late Q3 or Q4. 
<img width="984" alt="Screen Shot 2022-09-12 at 8 35 48 AM" src="https://user-images.githubusercontent.com/62632537/189655014-daa45e4a-0f5f-4d5a-abe8-e48a0ac3bd13.png">

[Figma](https://www.figma.com/file/2L3Fqq7D2iUQWbS1B1uqTJ/Connections?node-id=43%3A19663)
